references/classification/utils.py

```diff
@@ -409,3 +409,11 @@ def reduce_across_processes(val):
     dist.barrier()
     dist.all_reduce(t)
     return t
+
+
+def master_params(optimizer):
```

Comment:
naming nit: maybe `optimizer_params` or `get_optimizer_params`? I don't know why there is a "master"